content/docs/tutorials/get-started/data-versioning.md

```diff
@@ -251,10 +164,86 @@ $ dvc pull
 > [Sharing Data and Model Files](/doc/use-cases/sharing-data-and-model-files)
 > for more on basic collaboration workflows.
 
-## Other ways to track data
+## Making changes
+
+When you make a change to a file or directory, run `dvc add` again to track
+changes:
+
+```dvc
+$ dvc add data/data.xml
+```
+
+<details>
+
+### ⚙️ Expand to make some changes
+
+For the sake of simplicity let's just double the dataset artificially (and
+pretend that we got more data from some external source):
+
+```dvc
+$ cp data/data.xml /tmp/data.xml
+$ cat /tmp/data.xml >> data/data/xml
+```
+
+</details>
+
+Usually you would do `git commit` and `dvc push` to save the changes:
```

Comment:
```suggestion
Usually you would also `git commit` and `dvc push` to save the changes:
```

Reply:
Put inside details section though?

Reply:
I think it's fine to remind about Git importance from time to time